### PR TITLE
Mes 5145 catd

### PR DESCRIPTION
**Pull request by naqvis5**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/features/cat-d/02-Failed-journy.feature [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@catd
+
+  Feature: Driver Examiner failed the test journy for category D with serious fault
```
Comment (turnerc1) [inline test/e2e/features/cat-d/02-Failed-journy.feature]: Journey is spelt wrong here

**Review by turnerc1 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -224,4 +224,4 @@
       "npm run lint:fix"
     ]
   }
-}
+}
```
Comment (Cullima) [inline package.json:227]: can you revert this change

File: test/e2e/features/cat-d/01-pass-journy.feature [hunk 3]
```diff
@@ -0,0 +1,36 @@
+@catd
```
Comment (Cullima) [inline test/e2e/features/cat-d/01-pass-journy.feature]: typo in file name 'journy' should be 'journey'

File: test/e2e/features/cat-d/02-Failed-journy.feature [hunk 4]
```diff
@@ -0,0 +1,50 @@
+@catd
```
Comment (Cullima) [inline test/e2e/features/cat-d/02-Failed-journy.feature]: same typo in file name

**Review by Cullima — CHANGES_REQUESTED** [review]
Couple of small changes needed

File: test/e2e/features/cat-d/01-pass-journy.feature (hunk 3 above)
Comment (naqvis5) [reply test/e2e/features/cat-d/01-pass-journy.feature]: @Cullima  Thank you typo error has been fixed.

File: test/e2e/features/cat-d/02-Failed-journy.feature (hunk 4 above)
Comment (naqvis5) [reply test/e2e/features/cat-d/02-Failed-journy.feature]: @Cullima Thank you typo error has been fixed.

File: package.json (hunk 2 above)
Comment (naqvis5) [reply package.json:227]: I have reverted back but i don't know why it is showing space here. 227 is the last line in my json file with }

**Review by Cullima — APPROVED** [review]
